scripts/webui.py

```diff
@@ -72,6 +72,7 @@
 parser.add_argument('--horde_priority_usernames',type=str, action='append', required=False, help="Usernames which get priority use in this horde instance. The owner's username is always in this list.")
 parser.add_argument('--horde_max_power',type=int, required=False, help="How much power this instance has to generate pictures. Min: 2")
 parser.add_argument('--horde_nsfw', action='store_true', required=False, help="Set to false if you do not want this worker generating NSFW images.")
+parser.add_argument('--horde_blacklist', nargs='+', required=False, help="List the words that you want to blacklist.")
```

Comment:
please switch to string and append as in the AI horde

Reply:
This works with seperating the words by spaces.